### PR TITLE
Transform SetDisplayLayout requests into UI.Show for HMIs

### PR DESCRIPTION
**Pull request by ShobhitAd**
This proposal is to make modifications in sdl_core to transform incoming `SetDisplayLayout` requests from applications into `UI.Show` requests to send to the HMI.

### CODE REVIEW

**Comment by ShobhitAd** [comment]
@theresalech Ready for review

File: proposals/nnnn-transform-setdisplaylayout-requests-to-ui-show.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+## Motivation
+
+The `SetDisplayLayout` RPC was deprecated as a part of the changes for the [Widget Support](https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0216-widget-support.md#setdisplaylayout) feature. In that proposal, a new `templateConfiguration` parameter was added to the `Show` RPC to allow it to take over the responsibilities of `SetDisplayLayout`. Given that `UI.Show` can now handle switching the display layout and changing the color schemes in the HMI, it would make sense to use `UI.Show` instead of having multiple implementations for the same functionality.
+
+## Proposed solution
+
+The proposed solution is to transform incoming `SetDisplayLayout` requests into `UI.Show` requests to be sent to the HMI. This will require the following changes to `set_display_layout_request.cc`
+
+#### Change 1: Construct UI.Show request from SetDisplayLayout params
```
Comment (theresalech) [inline proposals/nnnn-transform-setdisplaylayout-requests-to-ui-show.md:20]:
```suggestion
#### Change 1: Construct `UI.Show` request from `SetDisplayLayout` params
```

File: proposals/nnnn-transform-setdisplaylayout-requests-to-ui-show.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+A `SetDisplayLayout` request can be transformed into a `UI.Show` request by using the `displayLayout`,`dayColorScheme`, `nightColorScheme` parameters from the original request in the `templateConfiguration` struct for the `UI.Show` request
```
Comment (theresalech) [inline proposals/nnnn-transform-setdisplaylayout-requests-to-ui-show.md:22]:
```suggestion
A `SetDisplayLayout` request can be transformed into a `UI.Show` request by using the `displayLayout`,`dayColorScheme`, `nightColorScheme` parameters from the original request in the `templateConfiguration` struct for the `UI.Show` request.
```

**Review by theresalech — CHANGES_REQUESTED** [review]
Hi @ShobhitAd - I've left two minor suggested changes. Please let me know if you have any questions. Thanks!

**Review by theresalech — APPROVED** [review]

**Pull request merged**